intersect.py

```diff
@@ -0,0 +1,94 @@
+3350. Intersection of Two Arrays II
+# Time Complexity : O(max(m,n)) where m is length of nums1, n is length of nums2
+# Space Complexity : O(n)
+# Did this code successfully run on Leetcode: Yes
+# Any problem you faced while coding this : No
+#Hashmap
+class Solution:
+    def intersect(self, nums1: List[int], nums2: List[int]) -> List[int]:
+        if not nums1 or not nums2:
+            return []
+        if len(nums1) > len(nums2):
+            temp = nums1
+            nums1 = nums2
+            nums2 = temp
+        mapp = {}
+        for i in range(len(nums1)):
+            if nums1[i] in mapp:
+                mapp[nums1[i]] += 1
+                #print(nums1[i])
+            else:
+                mapp[nums1[i]] = 1
+             
+        res = []
+        for i in range(len(nums2)):
+            if nums2[i] in mapp:
+                res.append(nums2[i])
+                mapp[nums2[i]] -= 1
+                if mapp[nums2[i]] == 0:
+                    del mapp[nums2[i]]
+        return res
+# Time Complexity : O(m+n) where m is length of nums1, n is length of nums2

```

Comment:
Please check the time complexity here. You are doing the sorting. So could be the time complexity?